runtime/testy/fake_dawn.py

```diff
@@ -2,34 +2,50 @@
 import threading
 import queue
 import time
+import runtime_pb2
+import ansible_pb2
 
 data = [0]
 send_port = 1236
 recv_port = 1235
 dawn_hz = 10
 
+def dawn_packager(data):
+    proto_message = ansible_pb2.DawnData()
+    proto_message.student_code_status = 1
```

Comment:
Umm, what???

This change is for human readability. It's a rule of good style to always use the name of an enum, not its number, when hardcoding constants.